async.md

```diff
@@ -220,4 +220,65 @@ Example
 		})
 ~~~
 
+## ClearBladeAsync.Collection.createIndex(columnToIndex)
+
+This function creates an index on a collection column.
```

Comment:
I think you can mention, by default, it's indexed on the item_id column. User can index the collection on any column of their choice.